UnitTests/CardPaymentsTests/ConfirmPaymentSourceRequest_Tests.swift

```diff
@@ -73,4 +73,39 @@ class ConfirmPaymentSourceRequest_Tests: XCTestCase {
         XCTAssertEqual(confirmPaymentSourceRequest.method, expectedMethod)
         XCTAssertEqual(confirmPaymentSourceRequest.headers, expectedHeaders)
     }
+
+    enum TestError: Error {
+        case encodeError
+    }
+    
+    class FailingJSONEncoder: JSONEncoder {
+        
+        override func encode<T>(_ value: T) throws -> Data where T: Encodable {
+            throw TestError.encodeError
+        }
+    }
+
+    func testEncodingFailure_throws_EncodingError() throws {
+        let mockOrderID = "mockOrderID"
+        let card = Card(
+            number: "4032036247327321",
+            expirationMonth: "11",
+            expirationYear: "2024",
+            securityCode: "222"
+        )
+        let cardRequest = CardRequest(orderID: mockOrderID, card: card)
+        
+        let failingEncoder = FailingJSONEncoder()
+        
+        XCTAssertThrowsError(try ConfirmPaymentSourceRequest(
+            accessToken: "fake token",
+            cardRequest: cardRequest,
+            encoder: failingEncoder)) { error in
```

Comment:
I believe this spacing should throw some sort of warning. We should update it to:
```swift
        XCTAssertThrowsError(
            try ConfirmPaymentSourceRequest(
                accessToken: "fake token",
                cardRequest: cardRequest,
                encoder: failingEncoder
            )
        ) { error in
```

Reply:
Addressed here: 7f57a8067d0624219df2ff69d2edf49deaf397c5